tools/rna_tools/miRDeep2/miRDeep2_quantifier/quantifier.xml

```diff
@@ -1,17 +1,20 @@
 <tool id="rbc_mirdeep2_quantifier" name="MiRDeep2 Quantifier" version="2.0.0">
-    <description>
-<![CDATA[
-fast quantitation of reads mapping to known miRBase precursors
-]]>
-    </description>
+    <description>fast quantitation of reads mapping to known miRBase precursors</description>
     <requirements>
-        <requirement type="package" version="2.0">mirdeep2_quantifier</requirement>
-        <requirement type="package" version="0.12.7">bowtie</requirement>
-        <requirement type="package" version="5.18.1">perl</requirement>
-        <requirement type="package" version="1.8.5">vienna_rna</requirement>
-        <requirement type="package" version="2.023">pdf_api2</requirement>
+        <requirement type="package" version="2.0.0.8">mirdeep2</requirement>
```

Comment:
bowtie and everything else is already defined as dependency here: https://github.com/bioconda/bioconda-recipes/blob/53f88dc6893c74cf170024401eaf62650a92ead3/recipes/mirdeep2/2.0.0.8/meta.yaml
So you don't need to add it again here.